apps/meteor/app/models/server/models/Users.js

```diff
@@ -1645,6 +1645,10 @@ Find users to send a message by email if:
 			},
 		});
 	}
+
+	countRoomMembers(roomId) {
+		return this.find({ __rooms: roomId, active: true }).count();
```

Comment:
shouldn't this be: `__rooms: { $in: [roomId] }`?

Reply:
For simple array of strings, it can be done like that:
https://www.mongodb.com/docs/manual/tutorial/query-arrays/